src/docusign/send-email-sign.ts

```diff
@@ -150,7 +150,10 @@ function htmlPage(args) {
 
         <h1 style="font-family: 'Trebuchet MS', Helvetica, sans-serif;
             color: darkblue;margin-bottom: 0;">${args.submitterName}: ${args.submitterEmail}</h1>
-  
+        
+        <h3 style="font-family: 'Trebuchet MS', Helvetica, sans-serif;
```

Comment:
Same as on the GUI - if there was more than 1 DARPA project, should this be in the DARPA row in the table?

Reply:
Ya, I had that thought. As this is based on the user allocation and that is not setup for multiple projects yet. Something for down the drag.